homeassistant/components/heatzy/climate.py

```diff
@@ -0,0 +1,295 @@
+"""Climate sensors for Heatzy."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from heatzypy.exception import HeatzyException
+
+from homeassistant.components.climate import (
+    ATTR_TARGET_TEMP_HIGH,
+    ATTR_TARGET_TEMP_LOW,
+    PRESET_AWAY,
+    PRESET_COMFORT,
+    PRESET_ECO,
+    PRESET_NONE,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import HeatzyDataUpdateCoordinator
+from .const import (
+    CFT_TEMP_H,
+    CFT_TEMP_L,
+    CONF_ALIAS,
+    CONF_ATTR,
+    CONF_ATTRS,
+    CONF_MODE,
+    CONF_MODEL,
+    CONF_ON_OFF,
+    CONF_PRODUCT_KEY,
+    CONF_VERSION,
+    CUR_TEMP_H,
+    CUR_TEMP_L,
+    DOMAIN,
+    ECO_TEMP_H,
+    ECO_TEMP_L,
+    ELEC_PRO_SOC,
+    GLOW,
+    PILOTEV1,
+    PILOTEV2,
```

Comment:
Should these also have underscores? (Or be named better - they seem to be lists of some sort of product IDs..? (Looking further, ditto with `GLOW` and `ELEC_PRO_SOC`.))